test/src/e2e_vm_tests/test_programs/should_pass/language/logging/json_abi_oracle.json

```diff
@@ -12,63 +12,63 @@
   ],
   "loggedTypes": [
     {
-      "logId": 165743,
+      "logId": 165753,
```

Comment:
You will no longer need these after https://github.com/FuelLabs/sway/pull/3083

Reply:
That is very good to hear :)